src/channel.ts

```diff
@@ -1300,12 +1299,13 @@ export class Channel<StreamChatGenerics extends ExtendableGenerics = DefaultGene
               } else {
                 channelState.read[userId].unread_messages += 1;
               }
+              if (userId === ownUserId && channelState.read[ownUserId].unread_messages === 1) {
```

Comment:
Can you please add a comment here @MartinCupela ? It's a bit hard to understand

Reply:
Added a comment.